### PR TITLE
V1.21.1

### PR DESCRIPTION
**Pull request by iliapolo**
See CHANGELOG

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.21.1](https://github.com/aws/aws-cdk/compare/v1.21.0...v1.21.1) (2020-01-16)
+
```
Comment (rix0rrr) [inline CHANGELOG.md]: Changelog should probably contain SOMETHING

Comment (eladb) [reply CHANGELOG.md]: Yeah. It’s because your commitS didn’t use conventional commits. Feel free to add by hand.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file. See [standa
 
 ## [1.21.1](https://github.com/aws/aws-cdk/compare/v1.21.0...v1.21.1) (2020-01-16)
 
+### Bug Fixes
+
+* **ecr-assets:** fix assembly creation to not include absolute paths to docker files ([#5807](https://github.com/aws/aws-cdk/issues/5807)) ([6bc8ecc](https://github.com/aws/aws-cdk/pull/5833/commits/6bc8eccdba21a10944c81ababe07df37c07481d0))
+* **cli:** revert cdk execution from inner directories ([#5826](https://github.com/aws/aws-cdk/issues/5826)) ([da9c626](https://github.com/aws/aws-cdk/pull/5833/commits/da9c6262fd262a5cfd8c150c2c165e46ecbc95b1))
```
Comment (eladb) [inline CHANGELOG.md]: “...due to an issue which will be fixed in a subsequent version”

Comment (iliapolo) [reply CHANGELOG.md]: 👍

**Comment by iliapolo** [comment]
Closing in favor of https://github.com/aws/aws-cdk/pull/5836 due to branch name change.

**Pull request closed without merging**